<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Playlist"/>
    <addaction name="actionAdd_Playlist"/>
    <addaction name="actionSave_Playlist"/>
    <addaction name="separator"/>
    <addaction name="actionNew_Node"/>
   </widget>
   <widget class="QMenu" name="menuLibrary">
    <property name="title">
     <string>Library</string>
    </property>
    <addaction name="actionPopulate_database"/>
    <addaction name="actionClear_database"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuSelect_theme">
     <property name="title">
      <string>Select theme</string>
     </property>
     <addaction name="actionBrowseForTheme"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="actionSet_playing_node_as_root"/>
    <addaction name="actionParty_mode"/>
    <addaction name="actionShuffle_mode"/>
    <addaction name="separator"/>
    <addaction name="menuSelect_theme"/>
    <addaction name="actionUpdate_theme"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionShortcuts"/>
    <addaction name="actionShow_help"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuLibrary"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string extracomment="comment">status bar</string>
   </property>
  </widget>
  <action name="actionNew_Node">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>New Node</string>
   </property>
  </action>
  <action name="actionCreate_database">
   <property name="text">
    <string>Create database</string>
   </property>
  </action>
  <action name="actionPopulate_database">
   <property name="text">
    <string>Add folder</string>
   </property>
  </action>
  <action name="actionDelete_database">
   <property name="text">
    <string>Delete database</string>
   </property>
  </action>
  <action name="actionClear_database">
   <property name="text">
    <string>Clear library</string>
   </property>
  </action>
  <action name="actionSet_playing_node_as_root">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Set playing circle as root</string>
   </property>
   <property name="statusTip">
    <string>Set playing circle as root: instead of forming trees of circles, always have just one circle and it's children.</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionUpdate_theme">
   <property name="text">
    <string>Update theme</string>
   </property>
  </action>
  <action name="actionBrowseForTheme">
   <property name="text">
    <string>Browse for theme</string>
   </property>
  </action>
  <action name="actionSave_Playlist">
   <property name="text">
    <string>Save Playlist</string>
   </property>
  </action>
  <action name="actionLoad_Playlist">
   <property name="text">
    <string>Load Playlist</string>
   </property>
  </action>
  <action name="actionAdd_Playlist">
   <property name="text">
    <string>Add Playlist (load without clearing)</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionShortcuts">
   <property name="text">
    <string>Shortcuts (keyboard, mouse)</string>
   </property>
  </action>
  <action name="actionShuffle_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Shuffle mode</string>
   </property>
   <property name="toolTip">
    <string>Shuffle mode</string>
   </property>
   <property name="statusTip">
    <string>Shuffle mode: stops getting related of playing tracks and keeps playing forever the existing circles you have.</string>
   </property>
  </action>
  <action name="actionShow_help">
   <property name="text">
    <string>Show help</string>
   </property>
  </action>
  <action name="actionParty_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Party mode</string>
   </property>
   <property name="statusTip">
    <string>Party mode: uses an extra trick or two to prevent music from stopping!</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
